src/vulkan-renderer/wrapper/device.cpp

```diff
@@ -440,6 +453,19 @@ Device::~Device() {
     vkDestroyDevice(m_device, nullptr);
 }
 
+bool Device::is_presentation_supported(const VkSurfaceKHR surface, const std::uint32_t queue_family_index) const {
+    // Default to true in this case where a surface is not passed (and therefore presentation isn't cared about)
+    VkBool32 supported = VK_TRUE;
+    if (surface != nullptr) {
```

Comment:
Why would `surface` be null here?

Reply:
Ooh I just copied that from `build_device_info`. It does not apply here!

Reply:
I removed it. Done.